templates/aurorapostgresql/template.yaml

```diff
@@ -0,0 +1,1225 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'AWS Service Broker - Amazon Aurora Postgresql.'
+
+###############################################################################
+# MetaData - ServiceBroker 
+###############################################################################   
+Metadata:
+  AWS::ServiceBroker::Specification:
+    Version: 1.0
+    Tags:
+    - AWS    
+    - Aurora
+    - PostgreSQL
+    - database
+    Name: rdspostgresql
```

Comment:
this must be changed to match the folder name `aurorapostgresql`

Reply:
changed it to match the folder name aurorapostgresql